motion_tracker/utils/crop_vis.py

```diff
@@ -27,8 +27,8 @@ def show_stages_of_random_crop(img, box_coords, output_width=256, output_height=
                                                   output_width, output_height)
     show_img(final_img, final_box_coords.as_array())
 
-def show_img(img, box_coords=None, window_name="Happy Dance Image", 
-                      msec_to_show_for=1500):
+def show_img(img, box_coords=None, window_name="Happy Dance Image",
+                      msec_to_show_for=1500, coords_are_fracs=False):
     """Show an image, potentially with surrounding bounding boxes
```

Comment:
Out of curiosity, is this giving us the ability to basically plot without having to un-normalize the predictions after training?